packages/node_modules/cerebral/src/operators/increment.js

```diff
@@ -5,42 +5,27 @@ export default function(target, value = 1) {
         'Cerebral operator.increment: You have to use the STATE or PROPS TAG as first argument'
       )
     }
+    let targetValue = resolve.value(target)
+    if (!Number.isInteger(targetValue)) {
```

Comment:
I thought the same :) but we don't need to put it into utils unless we need it in the other files. I would suggest keep it in this file. 
Btw. I am closing this PR because it does not support deep props setting. I will fix it in #960.